lib/msal-browser/src/config/Configuration.ts

```diff
@@ -57,6 +57,8 @@ export type BrokerOptions = {
     preferredInteractionType?: InteractionType.Popup | InteractionType.Redirect | InteractionType.None;
     allowBrokering?: boolean;
     trustedBrokerDomains?: string[];
+    brokerRedirectStartPage?: string;
```

Comment:
Any reason a brokered scenario would not want to use the same `redirectStartPage` and `onRedirectNavigate` as the non-brokered scenario?

Reply:
`redirectStartPage` and `onRedirectNavigate` shouldn't be set by the embedded application, since they are not the ones doing the redirect. The broker app should specify A. which page the redirect starts on and B. what happens after a redirect.

Reply:
We block all redirect APIs in an embedded scenario anyway, so these config options don't get used today in an embedded scenario.

Reply:
Sorry should have been more clear, why do we need separate `redirectStartPage` and `brokerRedirectStartPage` params instead of having the broker just use `redirectStartPage` regardless of whether it is getting tokens for itself or for an embedded app?

Reply:
redirectStartPage is set in the context of the client making the request, and is only a parameter on the request. The broker doesn't have any control over what the embedded application sends as the redirectStartPage, but we don't want the embeddedApp telling the broker to use the redirectStartPage for its own app - the broker should specify that themselves. The easiest way to do that is to add a broker config option for it in the constructor config.

Reply:
Got it. Forgot the params are only set on the request. Makes sense, thanks for clarifying